vernac/vernacexpr.ml

```diff
@@ -237,7 +237,7 @@ type vernac_cumulative = VernacCumulative | VernacNonCumulative
 
 (** {6 The type of vernacular expressions} *)
 
-type vernac_implicit_status = Implicit | MaximallyImplicit | NotImplicit
+type vernac_implicit_status = Impargs.implicit_kind = Implicit | MaximallyImplicit | NotImplicit
```

Comment:
Please deprecate this: add `[@ocaml.deprecated]` after each constructor and `[@@ocaml.deprecated]` after the whole declaration.


Or just remove it if you prefer, we give no guarantees of stability in the ml API.